common/src/dev-container-cli.ts

```diff
@@ -185,19 +185,17 @@ export interface DevContainerCliUpArgs {
   workspaceFolder: string;
   additionalCacheFroms?: string[];
   skipContainerUserIdUpdate?: boolean;
-  env?: string[];
   userDataFolder?: string;
+  additionalMounts?: string[];
 }
 async function devContainerUp(
   args: DevContainerCliUpArgs,
   log: (data: string) => void,
 ): Promise<DevContainerCliUpResult | DevContainerCliError> {
-  const remoteEnvArgs = getRemoteEnvArray(args.env);
```

Comment:
Is this an intentional removal?

Reply:
Lines 188, 195, and 200 in common/src/dev-container-cli.ts, I did not remove and this was from a clean rebase so I thought I should push first with them removed to ask. Should I add these lines back? The language server was indicating they were unused to me.

Reply:
Strange - they are present on latest `main` code: https://github.com/devcontainers/ci/blob/main/common/src/dev-container-cli.ts#L195

Reply:
Indeed